File: assets/js/volumeofSphere.js
// Problems: Volume of sphere, where the value of radius is given by the user.

/** Area of Rectangle formula 
 *  V = (4/3)*π*r³;
*/

const PI = 3.14159;

// Get from user
let r = +prompt('Enter the radius value: ');

// Volume of sphere equation
// V = (4/3)*π*r³;
const V = (4 / 3) * PI * (r * r * r);

// Display the result
console.log(`Radius value: ${r}`);
console.log(`Volume of sphere: ${V.toFixed(2)}`);

File: assets/c/volume-of-sphere.c
// Problems: Volume of sphere, where the value of radius is given by the user.

/** Volume of sphere formula 
 *  A = (4/3)πr³;
*/

#include <stdio.h>
#define PI 3.14159

int main() {
  float r, A;
  
  printf("Enter the radius value: ");
  scanf("%f", &r);

  // Volume of sphere
  A = (4 / 3) * PI * (r * r * r);

  // Display the volume of sphere
  printf("Volume of sphere is = %.2f ", A);

  return 0;
}